app/uk/gov/hmrc/minorentityidentificationfrontend/services/AuditService.scala

```diff
@@ -35,60 +35,69 @@ class AuditService @Inject()(auditConnector: AuditConnector,
       journeyConfig <- journeyService.getJourneyConfig(journeyId, authInternalId)
       optUtr <- storageService.retrieveUtr(journeyId)
       optOverseasTaxIdentifiers <- storageService.retrieveOverseasTaxIdentifiers(journeyId)
-    } yield journeyConfig.businessEntity match {
-      case OverseasCompany =>
-        val optUtrBlock = optUtr match {
-          case Some(utr: Ctutr) => Json.obj("userCTUTR" -> utr.value, "cTUTRMatch" -> false)
-          case Some(utr: Sautr) => Json.obj("userSAUTR" -> utr.value, "sautrMatch" -> false)
-          case None => Json.obj()
+      optRegistrationStatus <- storageService.retrieveRegistrationStatus(journeyId)
+    } yield {
+      val optUtrBlock = optUtr match {
+        case Some(utr: Ctutr) => Json.obj("userCTUTR" -> utr.value, "cTUTRMatch" -> false)
+        case Some(utr: Sautr) => Json.obj("userSAUTR" -> utr.value, "sautrMatch" -> false)
+        case None => Json.obj()
+      }
+      val overseasIdentifiersBlock =
+        optOverseasTaxIdentifiers match {
+          case Some(overseas) => Json.obj(
+            "overseasTaxIdentifier" -> overseas.taxIdentifier,
+            "overseasTaxIdentifierCountry" -> overseas.country)
+          case _ => Json.obj()
         }
-        val overseasIdentifiersBlock =
-          optOverseasTaxIdentifiers match {
-            case Some(overseas) => Json.obj(
-              "overseasTaxIdentifier" -> overseas.taxIdentifier,
-              "overseasTaxIdentifierCountry" -> overseas.country)
-            case _ => Json.obj()
-          }
-        //TODO Hardcoding the Verification and RegisterAPI status for now.  Will be updated in a future story
-        val auditJson = Json.obj(
-          "businessType" -> "Overseas Company",
-          "etmpPartyType" -> "55",
-          "VerificationStatus" -> Json.obj("verificationStatus" -> "UNCHALLENGED"),
-          "RegisterApiStatus" -> Json.obj("registrationStatus" -> "REGISTRATION_NOT_CALLED")
-        ) ++ optUtrBlock ++ overseasIdentifiersBlock
+      val registrationStatusBlock =
```

Comment:
Following on from Liz's updates I would just use "not called" for now. If we ever get to implement registration we can update the audit functionality.